test/cmd/help.sh

```diff
@@ -6,23 +6,21 @@ os::test::junit::declare_suite_start "cmd/help"
 # This test validates the help commands and output text
 
 # verify some default commands
-os::cmd::expect_success 'openshift'
 os::cmd::expect_success 'kubectl'
 os::cmd::expect_success 'oc'
 os::cmd::expect_success 'oc ex'
-os::cmd::expect_success 'origin'
 
 # help for root commands must be consistent
-os::cmd::expect_success_and_text 'openshift' 'Application Platform'
+os::cmd::expect_failure_and_text 'openshift' 'Application Platform'
 os::cmd::expect_success_and_text 'oc' 'OpenShift Client'
 os::cmd::expect_success_and_text 'oc -h' 'Build and Deploy Commands:'
 os::cmd::expect_success_and_text 'oc -h' 'Other Commands:'
 os::cmd::expect_success_and_text 'oc policy --help' 'add-role-to-user'
 os::cmd::expect_success_and_not_text 'oc policy --help' 'Other Commands'
 os::cmd::expect_success_and_not_text 'oc -h' 'Options'
 os::cmd::expect_success_and_not_text 'oc -h' 'Global Options'
-os::cmd::expect_success_and_text 'oc types' 'Deployment Config'
-os::cmd::expect_success_and_text 'oc adm ca' 'Manage certificates'
+os::cmd::expect_failure_and_text 'oc types' 'Deployment Config'
```

Comment:
why does this fail now?

Reply:
cmd/types also makes use of the DefaultSubCommandRun helper: https://github.com/openshift/origin/blob/master/pkg/oc/cli/cmd/types.go#L235